embassy-stm32/build.rs

```diff
@@ -433,13 +433,14 @@ fn main() {
                     fn frequency() -> crate::time::Hertz {
                         unsafe { crate::rcc::get_freqs().#clk }
                     }
-                    fn enable() {
+                    fn enable_and_reset() {
                         critical_section::with(|_| {
                             #before_enable
                             #[cfg(feature = "low-power")]
                             crate::rcc::clock_refcount_add();
                             crate::pac::RCC.#en_reg().modify(|w| w.#set_en_field(true));
                             #after_enable
+                            #rst
```

Comment:
since this is now inside a CS, the CS inside the `rst` code is no longer needed.

Reply:
Fixed